docs/user-guide/environment-variables.md

```diff
@@ -80,7 +80,7 @@ These environment variables may or may not be available depending on what plugin
 
 | Name | Description |
 |------|-------|
-| SD_ARTIFACTS_DIR | Location of built/generated files |
+| SD_ARTIFACTS_DIR | Location of built/generated files. _Note: If the build does not complete successfully, the UI will only display artifacts that were generated prior to the end of the build. If some artifacts are not available in the UI, you can still access them in the Store API (e.g.: `https://store.screwdriver.cd/v1/builds/12345/ARTIFACTS/fileName.txt`). You will need to pass in a token along with the request. See the [API docs](./api.md) for more details._ |
```

Comment:
I am not clear what this line is conveying ? `If the build does not complete successfully, the UI will only display artifacts that were generated prior to the end of the build. ` Isn't this the case always ? Files in $SD_ARTIFACTS_DIR will be available when build ends, except for aborted builds.

Reply:
Can't we just say something about the teardown artifact step and forget about the rest. dunno why we want to add thing to this section at the first place

Reply:
Not sure
cc @adong

Reply:
removed that line